nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListFileTransfer.java

```diff
@@ -138,4 +144,19 @@ protected boolean isListingResetNecessary(final PropertyDescriptor property) {
     protected abstract FileTransfer getFileTransfer(final ProcessContext context);
 
     protected abstract String getProtocolName();
+
+    protected void validateAdjustedTimeWindow(ValidationContext validationContext, Collection<ValidationResult> results) {
+        if (
+            BY_ADJUSTED_TIME_WINDOW.getValue().equals(validationContext.getProperty(LISTING_STRATEGY).getValue())
```

Comment:
Would it make sense to add a negative case, e.g.: when it's not adjusted time window, the time adjustment should _not_ be set?

Reply:
Yeah, I'll add that.